File: ClinicaAPI/Models/Receita.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace ClinicaAPI.Models
{
    public class Receita
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int Id { get; set; }
        [Required]
        public string? Medicamento { get; set; }
        [Required]
        public string? Dosagem { get; set; }
        [Required]
        public string? InstrucoesUso { get; set; }
        [Required]
        public DateTime? DataEmissao { get; set; }
        public Medico? Medico { get; set; }
        [Required]
        public int IdMedico { get; set; }
    }

}

File: AppClinica/src/app/Classes/Receita.ts
import { Medico } from "./Medico";

export class Receita {
    id: number = 0;
    medicamento: string = '';
    dosagem: string = '';
    instrucoesUso: string = '';
    dataEmissao: Date | null = null;
    medico: Medico | undefined;
    idMedico: number = 0;
}
